letsencrypt/cli.py

```diff
@@ -636,8 +638,12 @@ def preprocess_args(self, args):
                 self.verb = token
                 return reordered
 
-        self.verb = "run"
-        return args + ["run"]
+        if "--standalone" in args and "--installer" not in args and "-i" not in args:
```

Comment:
@bmw I think the version I implemented is a bit more elegant, but I don't feel very strongly about it.  Will pull out the default-changing logic.